docs/interact/programmability/lua-debugging.md

```diff
@@ -8,25 +8,25 @@ aliases:
   - /docs/manual/programmability/lua-debugging/
 ---
 
-Starting with version 3.2 Redis includes a complete Lua debugger, that can be
-used in order to make the task of writing complex Redis scripts much simpler.
+Starting with version 3.2 Valkey includes a complete Lua debugger, that can be
```

Comment:
Valkey 3.2 doesn't exist and it's ancient history. I think we can drop lines 11 and 12.